clothcalc_placeholder.js

```diff
@@ -7799,13 +7807,26 @@
                         eval("MarketWindow.getClearName = " + save.MarketWindow)
                     }
                 };
+
+                _self.injectGetBids = function() {
+		    if (typeof save.MarketWindowTab == "undefined") {
```

Comment:
Are you sure? This method of patching should stack just fine (the old function is preserved in a different variable and the new one calls it transparently before/after doing its own thing). I don't have time to look at the old code right now, but maybe you could just try?